src/TimeGutter.jsx

```diff
@@ -1,39 +1,53 @@
 import React from 'react';
-import cn from 'classnames';
 import dates from './utils/dates';
-import localizer from './localizer'
+
+import TimeSliceGroup from './TimeSliceGroup.jsx'
 
 let TimeGutter = React.createClass({
 
   propTypes: {
     step: React.PropTypes.number.isRequired,
+    slices: React.PropTypes.number,
+    now: React.PropTypes.instanceOf(Date).isRequired,
     min: React.PropTypes.instanceOf(Date).isRequired,
-    max: React.PropTypes.instanceOf(Date).isRequired
+    max: React.PropTypes.instanceOf(Date).isRequired,
+    showlabels: React.PropTypes.bool,
+    timesliceClassnames: React.PropTypes.string,
+    culture: React.PropTypes.string,
+    timeGutterFormat: React.PropTypes.string
+  },
+
+  getDefaultProps() {
+    return {
+      step: 10,
```

Comment:
where is `10` coming from, I think this is being overwritten by the parent, I'd remove the default to avoid confusion